lib/inspec/metadata.rb

```diff
@@ -123,6 +130,13 @@ def valid?
       errors.empty? && unsupported.empty?
     end
 
+    def valid_version?(value)
+      Semverse::Version.new(value)
+      true
+    rescue Semverse::InvalidVersionFormat => _ex
```

Comment:
Since we're not using the exception object, we can remove the `=> _ex` from here.